src/index.js

```diff
@@ -226,4 +228,34 @@ function applySesTransforms(browserify) {
 
   browserify.transform(removeHtmlComment, { global: true })
   browserify.transform(changeImportString, { global: true })
+}
+
+function validateConfig(configOverride) {
+  if (typeof configOverride !== 'object') {
+    throw new Error("LavaMoat - Expected config override to be an object")
+  }
+  
+  if (!'resources' in Object.keys(configOverride)) {
+    throw new Error("LavaMoat - Expected label 'resources' for configuration key")
+  }
+
+  Object.entries(configOverride['resources']).forEach(([packageName, packageOpts], index) => {
+    if (index === 0 && packageName !== '<root>') {
+      throw new Error("LavaMoat - Expected package '<root>' as first resource")
+    }
+    const packageOptions = Object.keys(packageOpts)
+    const packageEntries = Object.values(packageOpts)
+    if (!packageOptions.every(packageOpt => packageOpt === 'globals' || packageOpt === 'packages')) {
+      throw new Error("LavaMoat - Unrecognized package options. Expected 'globals' or 'packages'")
+    }
+
+    packageEntries.forEach((entry) => {
+      Object.values(entry).forEach((value) => {
+        if (value !== true) {
```

Comment:
`'write'` is a valid value for globals. it allows packages to write to `global[key]`